### PR TITLE
Feed back Issues and Refactor

### PR DESCRIPTION
**Pull request by ecwillis**

### CODE REVIEW

File: lib/configure.js [hunk 1]
```diff
@@ -1,10 +1,93 @@
...
+          return cb(err);
+        }
+
+        async.map(files, (file, acb) => {
+          acb(null, jsyaml.safeLoad( fs.readFileSync(file, 'utf-8') ));
+        }, cb);
+      }); 
+    },
+    getEnvConfigs: (cb) => {
+      glob("conf/**/!(default*)", (err, files) => {
```
Comment (jgallen23) [inline lib/configure.js]: any reason we aren't using confi for this?

Comment (ecwillis) [inline lib/configure.js]: Yes.
Confi assumes that you are in an environment, so all the configurations are passed translated down into a single object.
The lambda function would need to know about the configurations for all the environments at the same time to choose which one it needs.

**Comment by jgallen23** [comment]
#1

**Comment by jgallen23** [comment]
need readme

File: lib/configure.js [hunk 2]
```diff
@@ -1,10 +1,93 @@
...
+const async = require('async');
+const _ = require('lodash');
 const glob = require('glob');
-const jsyaml = require('jsyaml');
+const jsyaml = require('js-yaml');
+
+
+module.exports = (functionList, opts, done) => {
+  
+  let envFiles = [];
```
Comment (dawnerd) [inline lib/configure.js]: You using eslint? ;)

File: README.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
 # node-lambda
 
-An `aws-sdk` wrapper for lambda functions and deployments.
+A CLI / library for working with AWS lambda. Allows you to process and deploy lambda functions to AWS.
```
Comment (jgallen23) [inline README.md:3]: what should the config file look like? how do I call this via node lib?

**Pull request merged**